pages/docs/data-modeling/attributes.mdx

```diff
@@ -56,3 +55,18 @@ SQL can reference other attributes by surrounding their `alias` in double curly
 ### Using joined columns in custom SQL
 
 See [Using Joined Columns in Custom SQL](./model-joins#using-joined-columns-in-custom-sql) for more info.
+
+## Attribute folders
+
+You can organize your attributes into folders to keep your data model tidy and easy to navigate.
+
+To organize your attributes into folders in a data model:
+
+1. Open the data model in the [Model builder](/docs/data-modeling).
+2. Select the attribute you want to move to a folder.
+3. In the attribute configuration pane, locate the `Folder` input field.
+4. From the dropdown, select the folder to which you want to move the attribute.
+5. To create a new folder, type the desired name into the search input and click the option to `Add` a new folder with that name.
```

Comment:
Nit: this isn't an extra step, it should be part of step 4 along the lines of "either select the folder .... or create a new folder by ...."